pkgs/development/python-modules/monai-label/default.nix

```diff
@@ -0,0 +1,180 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  pythonAtLeast,
+
+  # build:
+  ninja,
+  setuptools,
+
+  # dependencies:
+  torch,
+  bcrypt,
+  cachetools,
+  dicomweb-client,
+  einops,
+  expiring-dict,
+  expiringdict,
+  fastapi,
+  filelock,
+  girder-client,
+  google-auth,
+  httpx,
+  monai,
+  passlib,
+  pydantic,
+  pydantic-settings,
+  pydicom,
+  pydicom-seg,
+  pyjwt,
+  pynetdicom,
+  pynrrd,
+  python-dotenv,
+  python-multipart,
+  pyyaml,
+  requests,
+  requests-toolbelt,
+  schedule,
+  scikit-learn,
+  scipy,
+  shapely,
+  timeloop,
+  urllib3,
+  uvicorn,
+  watchdog,
+
+  # test
+  pytestCheckHook,
+  parameterized,
+}:
+
+buildPythonPackage rec {
+  pname = "monai-label";
+  version = "0.8.4";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9" || pythonAtLeast "3.12"; # uses `distutils`
+
+  src = fetchFromGitHub {
+    owner = "Project-MONAI";
+    repo = "MONAILabel";
+    rev = "refs/tags/${version}";
+    hash = "sha256-4XhL8zdYz33UIyIk9aih8GsFj2ktSt07sCKk0mgpXFw=";
+  };
+
+  postPatch = ''
+    patchShebangs monailabel/scripts/monailabel
+  '';
+
+  build-system = [
+    ninja
+    setuptools
+  ];
+
+  dependencies =
+    [
+      bcrypt
+      cachetools
+      dicomweb-client
+      einops
+      expiring-dict
+      expiringdict
+      fastapi
+      filelock
+      girder-client
+      google-auth
+      httpx
+      monai
+      passlib
+      pydantic
+      pydantic-settings
+      pydicom
+      pydicom-seg
+      pyjwt
+      pynetdicom
+      pynrrd
+      python-dotenv
+      python-multipart
+      pyyaml
+      requests
+      requests-toolbelt
+      schedule
+      scikit-learn
+      scipy
+      shapely
+      timeloop
+      torch
+      urllib3
+      uvicorn
+      watchdog
+    ]
+    ++ (
+      let
+        d = monai.optional-dependencies;
+      in
+      d.fire
+      ++ d.gdown
+      ++ d.ignite
+      ++ d.itk
+      ++ d.lmdb
+      ++ d.mlflow
+      ++ d.nibabel
+      ++ d.numpymaxflow
+      ++ d.openslide
+      ++ d.pillow
+      ++ d.psutil
+      ++ d.scikit-image
+      ++ d.tensorboard
+      ++ d.torchvision
+      ++ d.tqdm
```

Comment:
This is *way* too long. Maybe something like `lib.flatten (lib.attrValues monai.optional-dependencies)` if you want all of them?

Reply:
The latter should be fine

Reply:
Perfect, done.